.github/workflows/provider-updoc.yml

```diff
@@ -53,10 +59,22 @@ jobs:
           GOOGLE_APPLICATION_CREDENTIALS: sa.json
           PROVIDER_NAME: ${GITHUB_REPOSITORY#*/}
           VER_MAJOR_MINOR: v${GITHUB_REF#"refs/heads/release-"}
+          SUBPACKAGES: ${{ inputs.publish-pack }}
         run: |
           if [[ "${GITHUB_REF##*/}" == release-* ]]; then
-            echo "Publishing Docs for ${{ env.PROVIDER_NAME }}, ${{ env.VER_MAJOR_MINOR }}"
-            go run github.com/upbound/uptest/cmd/updoc@${{ env.UPTEST_VERSION }} upload --docs-dir=./docs --name=${{ env.PROVIDER_NAME }} --version=${{ env.VER_MAJOR_MINOR }} --bucket-name=bucket-marketplace-docs-production --cdn-domain=https://user-content.upbound.io
+            for s in $SUBPACKAGES; do
+              PROVIDER_PACKAGE_NAME="${{ env.PROVIDER_NAME }}-$s"
+              DOCS_DIR="./docs/family"
```

Comment:
nit: In a future iteration, we may rename the `docs/family` folders in the official provider repositories as `docs/config` because `config` is the name that we use internally for referring to the family's config package. This would further allow us to compute the `docs` folder from which to publish the docs as follows:
```bash
DOCS_DIR="./docs/${s}"
```
This would also eliminate the conditionals in this block. It would also make it easy for us to publish specific docs for new resource providers (if this is needed in the future).